lib/logdna.rb

```diff
@@ -50,7 +50,7 @@ def default_opts
       }
     end
 
-    def level=(value)
+    def update_level=(value)
```

Comment:
change this back to level

Reply:
@vilyapilya, `rubocop` failed - it says there are 2 `level` defined: one as a field and one as a method - for cleaner code, I had to make this change + there is interesting case: I couldn't find anything calling this method

Reply:
Rubocop is configurable. And it should be ignored. This is functionally vital.

Reply:
I did the same mistake in the beginning, This is the function override.,  The parent class has the function named level and this is the code for modifying its behavior.= so it can be used with logdna. So it must be "level"

Reply:
Hang on a second, I didn't see this Rubocop error before, but this warning is legit - you can't have both a method named `level` and an `attr_accessor :level`. @vilyapilya 's right though that we need this method, but the right solution is to rename the attribute, not rename this method. And probably not make it `attr_accessor` - we only want it to be changed via this method.